### PR TITLE
Add data sorting(ascending/descending) by the selected properties on media pages

### PR DESCRIPTION
**Pull request by VkoHov**
- Enhance `SortPopover` component
- Change sort logic in Metrics Explore and  Images Explore tables
- Add sorting control to Images Explore

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1132](https://github.com/aimhubio/aim/issues/1132)

File: aim/web/ui/src/pages/Metrics/components/MetricsTableGrid/MetricsTableGrid.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
 import moment from 'moment';
+import _ from 'lodash';
```
Comment (arsengit) [inline aim/web/ui/src/pages/Metrics/components/MetricsTableGrid/MetricsTableGrid.tsx:2]: Lets use lodash-es

File: aim/web/ui/src/pages/Metrics/components/Table/SortPopover/SortPopover.scss [hunk 2]
```diff
@@ -2,16 +2,50 @@
...
-}
+  max-height: 21rem;
+  padding: 0.5rem 1rem;
+  width: toRem(392px);
+  &__optionList {
+    max-height: 12rem;
+    overflow: auto;
+  }
+  &__label {
+    margin-bottom: 4px;
```
Comment (arsengit) [inline aim/web/ui/src/pages/Metrics/components/Table/SortPopover/SortPopover.scss:13]: lets replace with $space-xxxs

File: aim/web/ui/src/pages/Metrics/components/Table/SortPopover/SortPopover.tsx [hunk 3]
```diff
@@ -1,18 +1,21 @@
 import React from 'react';
+import classNames from 'classnames';
+import _ from 'lodash';
```
Comment (arsengit) [inline aim/web/ui/src/pages/Metrics/components/Table/SortPopover/SortPopover.tsx:3]: ^

**Review by arsengit — APPROVED** [review]

File: aim/web/ui/src/pages/ImagesExplore/components/ImagesExploreTableGrid/ImagesExploreTableGrid.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
 import moment from 'moment';
+import _ from 'lodash';
```
Comment (roubkar) [inline aim/web/ui/src/pages/ImagesExplore/components/ImagesExploreTableGrid/ImagesExploreTableGrid.tsx:2]: please use `lodash-es`

File: aim/web/ui/src/utils/getSortedFields.ts [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    case SortActionTypes.ORDER_CHANGE: {
+      if (!_.isNil(index)) {
+        sortFields[index].order = order || 'asc';
+      }
+      return [...sortFields];
+    }
+    case SortActionTypes.ORDER_TABLE_TRIGGER: {
+      if (index === -1) {
+        sortFields.push({ ...field, order: 'asc', readonly: false } as any);
```
Comment (roubkar) [inline aim/web/ui/src/utils/getSortedFields.ts:53]: please add types for `sortField`

**Review by roubkar — CHANGES_REQUESTED** [review]

File: aim/web/ui/src/services/models/imagesExplore/imagesExploreAppModel.ts [hunk 6]
```diff
@@ -1312,66 +1318,102 @@ function updateSortFields(sortFields: SortField[]) {
...
+      ...configData,
+      images,
+    };
+    model.setState({
+      config: configUpdate,
+    });
+
+    updateModelData(configUpdate, true);
+  }
+  analytics.trackEvent(
```
Comment (roubkar) [inline aim/web/ui/src/services/models/imagesExplore/imagesExploreAppModel.ts:1349]: @rubenaprikyan check this event please

**Review by roubkar — APPROVED** [review]

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Pull request merged**